app/src/main/res/layout/activity_main.xml

```diff
@@ -13,8 +13,16 @@
         android:onClick="onClickLaunchSignInView"
         android:text="Launch Sign In View"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintEnd_toEndOf="parent" />
+
+        <Button
+        android:id="@+id/main_forum_button"
```

Comment:
When I render the app on my computer I see that there is a problem with the constraints of the `LaunchSignInView`, if it's also the case for you could you add some basic constraints to the Forum button and sides for example?